UnityProject/Assets/Scripts/Core/Sprite Handler/SpriteHandler.cs

```diff
@@ -48,7 +48,10 @@ public class SpriteHandler : MonoBehaviour
 	[SerializeField] private List<Color> palette = new List<Color>();
 	public List<Color> Palette => palette;
 
-	private bool PaletteSet = false;
+	/// <summary>
+	/// false if the palette has not been configured for the current spriteSO. true otherwise
+	/// </summary>
+	private bool paletteSet = false;
```

Comment:
Since this is a serialized field and meant to be used from the unity inspector, replace the summary docstring for a Tooltip attribute.

Reply:
On my phone but if this is getting shown in the unity inspector, it shouldn't be... Will take a look later

Reply:
Oh silly me. I will add one to palette. Thanks for the review